gwsumm/tabs/stamp.py

```diff
@@ -94,7 +94,7 @@ def write_state_html(self, state):
         a = markup.oneliner.a('analysis', href=self.url+'/',
                               class_='alert-link', rel='external',
                               target='_blank')
-        if not(os.path.isdir(self.directory)):
+        if not (os.path.isdir(self.directory)):
```

Comment:
While you're at it, the outer parentheses should be removed as they have no purpose:

```suggestion
        if not os.path.isdir(self.directory):
```